mathics/builtin/assignment.py

```diff
@@ -1748,20 +1748,17 @@ def apply(self, module, evaluation):
         except ImportError as e:
             evaluation.message(self.get_name(), 'notfound', module)
             return SymbolFailed
-        except PyMathicsLoadException as e:
```

Comment:
Why was this block removed?

Reply:
Because it is better to catch it in the apply method of LoadModule, in order to be able of sending an error message using the method in evaluation.